mne/time_frequency/tests/test_tfr.py

```diff
@@ -133,11 +137,22 @@ def test_time_frequency():
     Fs = raw.info['sfreq']  # sampling in Hz
     tfr = cwt_morlet(data[0], Fs, freqs, use_fft=True, n_cycles=2)
     assert_true(tfr.shape == (len(picks), len(freqs), len(times)))
+    tfr2 = cwt_morlet(data[0], Fs, freqs, use_fft=True, n_cycles=2,
+                      decim=slice(0, 2))
+    assert_true(tfr2.shape == (len(picks), len(freqs), 2))
 
     single_power = single_trial_power(data, Fs, freqs, use_fft=False,
                                       n_cycles=2)
+    single_power2 = single_trial_power(data, Fs, freqs, use_fft=False,
+                                       n_cycles=2, decim=slice(0, 2))
 
     assert_array_almost_equal(np.mean(single_power), power.data)
+    # XXX cmoutard: does it make any sense to average single_power across
+    # all the dimensions and not just across epochs ? Because here we
+    # compare a numpy.float64 and a numpy.ndarray
+    # Below is a less conservative test for the decim argument.
```

Comment:
@agramfort Just to check, is this what you intended in https://github.com/mne-tools/mne-python/commit/09548d35848a5c34023b3280eedb7556cc6725f3?